docs/.docz/doczrc.js

```diff
@@ -0,0 +1,36 @@
+// https://www.docz.site/docs/project-configuration
+export default {
+  themeConfig: {
+    initialColorMode: 'dark'
```

Comment:
Until I have a better logo that works better in dark mode, let's start with light mode initially:

![image](https://user-images.githubusercontent.com/573625/90849947-81993380-e33e-11ea-82a9-39d427a21557.png)